src/mimi-core.jl

```diff
@@ -115,6 +117,20 @@ function variables(mi::ModelInstance, componentname::Symbol)
     return fieldnames(mi.components[componentname].Variables)
 end
 
+function isuniform(values::Vector)
```

Comment:
Could you add a comment above this that describes what this does? One short sentence is enough.